CHANGES.md

```diff
@@ -6,6 +6,10 @@ Change Log
 ##### Additions :tada:
 * Added optional `index` parameter to `PrimitiveCollection.add`. [#8041](https://github.com/AnalyticalGraphicsInc/cesium/pull/8041)
 
+##### Fixes :wrench:
+* Disable FXAA by default. [#7875](https://github.com/AnalyticalGraphicsInc/cesium/issues/7875)
```

Comment:
Provide a small snippet of code to show up to re-enable it (look at older changes in this file to see how we normally do this)